tests/foreman/longrun/test_inc_updates.py

```diff
@@ -16,279 +16,240 @@
-                self.qe_lce = LifecycleEnvironment(id=46).read()
-
-        :expectedresults: Incremental update completed with no errors and
-            Content view has a newer version
-
-        :CaseLevel: System
-        """
-        # Get the content view versions and use the recent one.  API always
-        # returns the versions in ascending order so it is safe to assume the
-        # last one in the list is the recent
-        cv_versions = self.rhel_6_partial_cv.version
-
-        # Get the applicable errata
-        errata_list = self.get_applicable_errata(self.rhva_6_repo)
-        self.assertGreater(len(errata_list), 0)
-
-        # Apply incremental update using the first applicable errata
-        entities.ContentViewVersion().incremental_update(
-            data={
-                'content_view_version_environments': [
-                    {
-                        'content_view_version_id': cv_versions[-1].id,
-                        'environment_ids': [self.qe_lce.id],
-                    }
-                ],
-                'add_content': {'errata_ids': [errata_list[0].id]},
-            }
-        )
-
-        # Re-read the content view to get the latest versions
-        self.rhel_6_partial_cv = self.rhel_6_partial_cv.read()
-        self.assertGreater(len(self.rhel_6_partial_cv.version), len(cv_versions))
-
-    @pytest.mark.tier4
-    @pytest.mark.upgrade
-    def test_positive_noapply_cli(self):
-        """Check if cli incremental update can be done without
-        actually applying it
-
-        :id: f25b0919-74cb-4e2c-829e-482558990b3c
-
-        :expectedresults: Incremental update completed with no errors and
-            Content view has a newer version
-
-        :CaseLevel: System
-        """
-        # Get the content view versions and use the recent one.  API always
-        # returns the versions in ascending order so it is safe to assume the
-        # last one in the list is the recent
-
-        cv_versions = self.rhel_6_partial_cv.version
-
-        # Get the applicable errata
-        errata_list = self.get_applicable_errata(self.rhva_6_repo)
-        self.assertGreater(len(errata_list), 0)
-
-        # Apply incremental update using the first applicable errata
-        ContentViewCLI.version_incremental_update(
-            {
-                'content-view-version-id': cv_versions[-1].id,
-                'lifecycle-environment-ids': self.qe_lce.id,
-                'errata-ids': errata_list[0].id,
-            }
-        )
-
-        # Re-read the content view to get the latest versions
-        self.rhel_6_partial_cv = self.rhel_6_partial_cv.read()
-        self.assertGreater(len(self.rhel_6_partial_cv.version), len(cv_versions))
+        # Add filter of type include but do not include anything.
+        # this will hide all RPMs from selected erratum before publishing.
+        entities.RPMContentViewFilter(
+            content_view=module_cv, inclusion=True, name="Include Nothing"
```

Comment:
Please switch these to single-quote strings.

Reply:
Done